### DISCUSSION TITLE
`permissions_for` on threads should take account of private threads

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [Rapptz/discord.py#9433] [Ideas]
For private threads, `permissions_for` does not indicate if a user can view the thread. Maybe it should take care of it.

**Comment by Rapptz** [comment]
It's not a permission (not even an implicit one) so it wouldn't make sense to, in my opinion. You also can't get private threads you don't have access to at all.

**Reply by PythonCoderAS** [reply]
Well is there a non-async way to check for thread membership? Right now I just loop over `thread.members` but it isn't ideal.

**Reply by Rapptz** [reply]
Discord doesn't reliably provide member information over the gateway so your best bet is using `fetch_member` and just swallowing it. Though I see there's an oversight and no `get_member`.
